src/pages/billing/Project.js

```diff
@@ -289,26 +287,21 @@ const ProjectDetail = ({ project, billingAccounts, authorizeAndLoadAccounts }) =
   })
 
   // Helpers
-  const updateBillingProject = withErrorReporting('Error updating billing project')(
-    () => Ajax(signal).Billing.billingProject(billingProject.projectName).then(setBillingProject)
+  const setBillingAccount = withErrorReporting('Error updating billing account')(
```

Comment:
I did have a spinner initally and it looked good when I was updating all the proejcts. When I made the change to update only one project, the spinner appeared and went so quickly that it looked like a weird rendering glitch so I removed it. What do you think

Reply:
There can be some perceived glitchiness with busy spinners, but when it comes to async calls I prefer to keep them in. We never know how someone's internet connection is, or whether the server may take an extra second to respond.

Reply:
They look shit though lol

Reply:
This is a very good point that I'd like to discuss further. In the past, I've had spinners with logic to only appear after a slight delay in case it turns out the operation completed quickly and, once rendered, stay for a minimum duration to avoid jitter. Would it be worth making this a pattern in this codebase?